### PR TITLE
Added: LearnWeb3-DAO

### PR DESCRIPTION
**Pull request by TreciaKS**
## Changes proposed

YouTube link for LearnWeb3-DAO channel in web3-metaverse.json file

## Screenshots
n/a
Checkout deployment

### CODE REVIEW

File: database/youtube/web3-metaverse.json [hunk 1]
```diff
@@ -33,12 +32,19 @@
...
         "name": "Francesco Cuilla",
         "description": "Francesco discusses a wide range of tech topics such as DevRel, Docker, Web3, and Blockchain.",
         "url": "https://www.youtube.com/@francescociulla",
         "category": "youtube",
         "subcategory": "web3-metaverse",
         "language": "english"
+      },
+     {
+        "name": "LearnWeb3 DAO",
+        "description": "A free program that will turn you into a web3 developer.",
```
Comment (rupali-codes) [inline database/youtube/web3-metaverse.json:44]: Hey Trecia, can we have a little longer desc?

Comment (TreciaKS) [reply database/youtube/web3-metaverse.json]: Done ✔

**Review by rupali-codes — APPROVED** [review]
Looks better ✨

**Comment by rupali-codes** [comment]
@TreciaKS could you please resolve merge conflicts :)

**Pull request closed without merging**